.codecov.yml

```diff
@@ -39,3 +39,4 @@ ignore:
   - "src/lib.rs"
   - "src/types.rs"
   - "src/state.rs"
+  - "packages/desmos-std/**/mock*.rs"
```

Comment:
Is there any reason to ignore it?

Reply:
Since are mocks I don't think is actually useful to include that code in the codecov report

Reply:
I think it's safe to ignore them since they're only used for testing

Reply:
Okay, got it. 👍